shepherd/DummyScoreboard.py

```diff
@@ -0,0 +1,9 @@
+import queue
+import LCM
+from Utils import *
```

Comment:
From now on, no wildcard imports for anything but the simplest modules. It pollutes the global namespace and obscures which modules names come from.